libs/pages/elewa/about-us/src/lib/team-members/membersInterface.ts

```diff
@@ -0,0 +1,41 @@
+export interface Members {
+  name: string;
+  role: string;
+  imageLink: string;
+  LinkedInProfileLink?: string;
+  summary: string;
+}
+export const testData = [
+  {
+    name: 'Jente Rosseel',
+    role: 'Founder, Managing Partner',
+    imageLink:
+      'https://images.unsplash.com/photo-1570295999919-56ceb5ecca61?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxzZWFyY2h8Mnx8bWFufGVufDB8fDB8fA%3D%3D&auto=format&fit=crop&w=700&q=60',
+    LinkedInProfileLink: '',
+    summary: 'lorem ipsum dolor sint',
+  },
+  {
+    name: 'Peter Reinartz',
+    role: 'Founder, Executive Chairman',
+    imageLink:
```

Comment:
Move members interface to libs/model/elewa (package)

name file elewa-team-members.interface.ts

export this file on this index.ts of this library for reuse in other places like this issue.